extensions/wikia/Email/Controller/FounderDigestController.class.php

```diff
@@ -0,0 +1,307 @@
+					'label' => 'Page edits',
+					'tooltip' => 'Number of edits made to the wiki by users'
+				]
+			]
+		];
+
+		return array_merge_recursive( $formFields, parent::getEmailSpecificFormFields() );
+	}
+}
+
+class FounderPageViewsDigestController extends FounderDigestController {
+	const TRACKING_CATEGORY_EN = TrackingCategories::FOUNDER_VIEWS_DIGEST_EN;
+	const TRACKING_CATEGORY_INT = TrackingCategories::FOUNDER_VIEWS_DIGEST_INT;
+
+	public function initEmail() {
+		$this->wikiId = $this->request->getVal( 'wikiId' );
+		$this->pageViews = $this->request->getVal( 'pageViews' );
+		$this->assertValidParams();
+
+		$this->pageViews = number_format( $this->pageViews );
+	}
+
+	protected function getSubject() {
+		return $this->getMessage( 'emailext-founder-views-digest-subject',
+			\WikiMap::getWikiName( $this->wikiId ) )->parse();
+	}
+
+	/**
+	 * @template digestLayout
+	 */
+	public function body() {
+		$this->response->setData( [
+			'salutation' => $this->getSalutation(),
+			'summary' => $this->getSummary(),
+			'buttonText' => $this->getButtonText(),
+			'buttonLink' => $this->getButtonLink(),
+			'details' => $this->getDetailsList(),
+			'hasContentFooterMessages' => false
+		] );
+	}
+
+	public function assertCanEmail() {
+		parent::assertCanEmail();
+		$this->assertNotSubscribedToCompleteDigest();
+		$this->assertSubscribedToPageViewsDigest();
+	}
+
+	/**
+	 * Checks if the founder is subscribed to the complete digest. If they are, don't send a page views
+	 * digest, which is included in the complete digest.
+	 *
+	 * @throws \Email\Check
+	 */
+	protected function assertNotSubscribedToCompleteDigest() {
+		if ( $this->targetUser->getBoolOption( 'founderemails-complete-digest-' . $this->wikiId ) ) {
+			throw new Check( 'Founder is subscribed to complete digest and should not receive page views digest.' );
+		}
+	}
+
+	/**
+	 * Asserts that the founder is subscribed to the page view digest.
+	 *
+	 * @throws \Email\Check
+	 */
+	public function assertSubscribedToPageViewsDigest() {
+		if ( !$this->targetUser->getBoolOption( 'founderemails-views-digest-' . $this->wikiId ) ) {
+			throw new Check( 'Founder is not subscribed to page views digest.' );
+		}
+	}
+
+	/**
+	 * Returns list of details for the digest
+	 *
+	 * @return array
+	 */
+	protected function getDetailsList() {
+		return [
+			[
+				'iconSrc' => \GlobalFile::newFromText( 'Page-views.png', \Wikia::NEWSLETTER_WIKI_ID )->getUrlGenerator()->url(),
+				'detailsHeader' => $this->getMessage( 'emailext-founder-digest-views-header', $this->pageViews )->parse(),
+				'details' => $this->getMessage( 'emailext-founder-digest-views-description-2' )->text()
+			] 
+		];
+	}
+
+	public function getButtonText() {
+		return $this->getMessage( 'emailext-founder-views-digest-link-label' )->text();
+	}
+
+	public function getButtonLink() {
+		return \GlobalTitle::newFromText( 'CreatePage', NS_SPECIAL, $this->wikiId )->getFullURL( array( 'modal' => 'AddPage' ) );
```

Comment:
So we made it a [Wikia coding convention](https://github.com/Wikia/guidelines/blob/master/PHP/CodingConventions.md#array-syntax) to use the new array syntax, ie, `[ 'modal' => 'AddPage' ]`